website/docs/migration.md

```diff
@@ -0,0 +1,244 @@
+---
+sidebar_position: 3
+---
+
+# Migrating from MongoDB to FerretDB
+
+To ensure a smooth and successful migration from MongoDB, we offer several methods through which you can test your application with FerretDB.
+
+## Operation modes
+
+We support different operation modes, by default FerretDB will always run on `normal` mode.
+In this mode all client requests are processed _only_ by FerretDB and returned to the client.
+
+You can set modes using either the `--mode` flag or the `FERRETDB_MODE` environment variable, accepting values such as `normal`, `proxy`, `diff-normal`, and `diff-proxy`.
```

Comment:
I think we should give a shorter overview and link to https://docs.ferretdb.io/configuration/operation-modes/ instead, with the rest of the page focusing on how to _use_ those modes, not what they _are_